### PR TITLE
fix grille shocking someone twice in a row

### PR DESCRIPTION
**Pull request by sp1d3rx**
bug # 1004

### CODE REVIEW

**Comment by XeonFaux** [comment]
Fixes #1004

File: code/game/objects/structures/grille.dm [hunk 1]
```diff
@@ -233,10 +234,15 @@
...
 	if(!in_range(src, user))//To prevent TK and mech users from getting shocked
 		return 0
+	if(hasShocked)
+		return 0	//Already shocked someone recently?
 	var/turf/T = get_turf(src)
 	var/obj/structure/cable/C = T.get_cable_node()
 	if(C)
 		if(electrocute_mob(user, C, src))
+			hasShocked = 1
+			spawn(10)
```
Comment (Kokojos) [inline code/game/objects/structures/grille.dm:244]: Let's say two people are running one behind the other and they get shocked, only the first will be shocked and the 2nd one will only bump it.
I suggest putting it lower, the bug was only happened nearly exactly the same time IRRC.

Comment (Pyko1) [inline code/game/objects/structures/grille.dm]: Or maybe even make it store the user it shocks for split second, and then check if the person getting shocked is the stored user?

**Comment by Nokington** [comment]
sure

**Comment by sp1d3rx** [comment]
it's ten milliseconds isnt it, i dont see a problem.

**Comment by TDcoolguy300** [comment]
Ten deciseconds, as BYOND time is handled in Deciseconds.

**Comment by sp1d3rx** [comment]
ok so 1 second

**Comment by Pyko1** [comment]
I assume this also fixes https://github.com/HippieStationCode/HippieStation13/issues/1232 ?

**Comment by nalarmg** [comment]
Coders/maintainers - safe to merge?

**Comment by danhibiki** [comment]
Yesssss

**Comment by sp1d3rx** [comment]
yes safe to merge, no it doesn't fix 1232

**Comment by ghost** [comment]
snowflake as fuck.

**Pull request merged**